tests/integration/PayoutCreateTest.php

```diff
@@ -29,11 +29,14 @@
         ->zip('EC2A 1PX')
         ->countryCode('GB');
 
+    $schemeSelection = $client->payoutSchemeSelection()->instantOnly();
```

Comment:
is this not already covered in the test `sends user selected scheme selection` (and ones below)?